R/derive_params_growth_height.R

```diff
@@ -57,7 +56,7 @@
 #'
 #'  The specified variables are set to the specified values for the new
 #'  observations. For example,
-#'   `set_values_to_sds(exprs(PARAMCD = “BMIASDS”, PARAM = “BMI-for-height z-score”))`
+#'   `set_values_to_sds(exprs(PARAMCD = “WTASDS”, PARAM = “Weight-for-height z-score”))`
```

Comment:
These examples should be H instead of A in the PARAMCDs - as the A stands for AGE, H for HEIGHT